build.gradle

```diff
@@ -16,10 +16,50 @@
  */
 
 plugins {
-  id "org.nosphere.apache.rat" version "0.3.0"
+  id 'org.nosphere.apache.rat' version '0.3.0'
+  id 'org.ajoberstar.grgit' version '1.6.0'
 }
 
-apply plugin: "org.nosphere.apache.rat"
+apply plugin: 'org.ajoberstar.grgit'
+
+task clean << {
+  delete buildDir
+}
+
+def dockerCmd = "docker run -v '${projectDir}':/geode-site -p 3000:3000 -w /geode-site/website apachegeode/geode-site-build "
+
+task compile(type:Exec, dependsOn: rat) {
+  inputs.dir "${projectDir}/website"
+  outputs.dir "${buildDir}/content"
+  executable 'bash'
+  args "-c", dockerCmd + "nanoc compile"
+}
+
+task view(type:Exec, dependsOn: compile) {
+  executable 'bash'
+  args "-c", dockerCmd + "nanoc view"
+}
+
+task publish(dependsOn: compile) {
```

Comment:
I thought about that, but I'm hesitant to go all the way (git add/commit/push) since that would require a clean workspace and we'd have to figure out `git rm` cases.  At least this can be used as a starting point and is less confusing than the prior `tar` instructions.